### PR TITLE
Feature: Markets, Marketdepth, Orderinput and PercantageButton compon…

### PR DESCRIPTION
**Pull request by Kohelbekker**
…ents added

### CODE REVIEW

File: src/components/MarketDepths/index.tsx [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+    AreaChart,
+    CartesianGrid,
+    ResponsiveContainer,
+    Tooltip,
+    TooltipPayload,
+    XAxis,
+    YAxis,
+} from 'recharts';
+
+// tslint:disable
```
Comment (josadcha) [inline src/components/MarketDepths/index.tsx]: remove please disabling of tslint

File: src/components/OrderInput copy/OrderInput.pcss [hunk 2]
```diff
@@ -0,0 +1,47 @@
+.cr-order-input {
```
Comment (josadcha) [inline src/components/OrderInput copy/OrderInput.pcss]: fix this file name

File: src/components/OrderInput copy/OrderInput.test.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import { mount, shallow } from 'enzyme';
```
Comment (josadcha) [inline src/components/OrderInput copy/OrderInput.test.tsx]: and this too

File: src/components/OrderInput copy/index.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
+import cr from 'classnames';
```
Comment (josadcha) [inline src/components/OrderInput copy/index.tsx]: +

File: src/components/OrderInput copy/readme.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+InputBlock example:
```
Comment (josadcha) [inline src/components/OrderInput copy/readme.md]: +

**Comment by josadcha** [comment]
unnecessary crypto icon in order inputs
image

File: src/components/OrderInput copy/OrderInput.pcss (hunk 2 above)
Comment (mod) [reply src/components/OrderInput copy/OrderInput.pcss]: I think this should be deleted

**Pull request merged**